src/FurCoNZ.Web/Helpers/DammAlgorithm.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace FurCoNZ.Web.Helpers
+{
+    /// <summary>
+    /// Based on https://en.wikipedia.org/wiki/Damm_algorithm
+    /// </summary>
+    public static class DammAlgorithm
```

Comment:
This is really cool! If you get some time, could we move this to an external .NET Standard NuGet package under the `InFurSecDen.Utils.*` namespace?

I was thinking of also porting it to inherit from [System.Security.Cryptography.HashAlgorithm](https://docs.microsoft.com/en-us/dotnet/api/system.security.cryptography.hashalgorithm?view=netcore-2.2) (in a similar style to [DamienGKit's Crc32 class](https://github.com/damieng/DamienGKit/blob/master/CSharp/DamienG.Library/Security/Cryptography/Crc32.cs)) but that expects the input to be a `byte[]`, rather than an integer. Probably fine just porting it as is in this case :3